poc/vdaf_poc/flp_bbcggi19.py

```diff
@@ -1147,3 +1109,65 @@ def decode(self, output: list[F], _num_measurements: int) -> int:
     def test_vec_set_type_param(self, test_vec: dict[str, Any]) -> list[str]:
         test_vec['max_measurement'] = int(self.max_measurement)
         return ['max_measurement']
+
+
+def encode_range_checked_int(
+        field: type[F],
+        value: int,
+        max_measurement: int) -> list[F]:
+    """
+    Encode an integer into multiple field elements in a way that
+    allows for efficient range proofs.
+
+    Pre-conditions:
+
+        - `value >= 0`
+        - `max_measurement > 0`
```

Comment:
nit: is `max_measurement == 0` a valid value? It's obviously useless, but I just wanted to check my understanding.

Reply:
The SumVec circuit constructor rejects a `max_measurement` of zero, for example. The construction doesn't make much sense with a maximum measurement of zero, because that can be represented with zero bits, but we want to reserve one of those zero bits for the last weight.